src/DefaultBuilder/src/PublicAPI.Unshipped.txt

```diff
@@ -1,8 +1,12 @@
 #nullable enable
 Microsoft.AspNetCore.Builder.Configuration
+Microsoft.AspNetCore.Builder.Configuration.AutoUpdate.get -> bool
+Microsoft.AspNetCore.Builder.Configuration.AutoUpdate.set -> void
 Microsoft.AspNetCore.Builder.Configuration.Configuration() -> void
+Microsoft.AspNetCore.Builder.Configuration.Dispose() -> void
 Microsoft.AspNetCore.Builder.Configuration.GetChildren() -> System.Collections.Generic.IEnumerable<Microsoft.Extensions.Configuration.IConfigurationSection!>!
 Microsoft.AspNetCore.Builder.Configuration.GetSection(string! key) -> Microsoft.Extensions.Configuration.IConfigurationSection!
+Microsoft.AspNetCore.Builder.Configuration.Update() -> void
```

Comment:
I can make these internal if that makes getting this into preview5 easier. I think we should add it though. I plan on adding these new APIs to my API proposal in the runtime repo: https://github.com/dotnet/runtime/issues/51770

Reply:
Make them all internal in preview5.

Reply:
I am going to make this internal. It's fairly likely the API will change anyway.